CMakeLists.txt

```diff
@@ -0,0 +1,610 @@
+cmake_minimum_required(VERSION 3.4.1)
+project(moai_dev)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_C_FLAGS "-DANDROID -DFT2_BUILD_LIBRARY -DHAVE_MEMMOVE=1 -DXML_STATIC")
```

Comment:
These flags are copied from the old build system in the Android.mk and build.sh files